_pages/home/updating-to-version-5/checkout-libraries-release-notes.md

```diff
@@ -223,6 +223,42 @@ Spartacus schematics are designed to assist you in your migration and to help ma
 
 Take a look at the [5.0 TypeScript Changes](https://github.com/SAP/spartacus/blob/84e6b462db1f5dca5c3145c5bc535fc5e2b52fe2/docs/migration/5_0-generated-typescript-changes-doc.md) to get a high-level overview of what has changed.
 
+### Configuring checkout that is already configured to load from the old checkout entrypoint '@spartacus/checkout/root' either eagerly or lazily.
+
+To re-iterate, if you are using Spartacus 4.0, the checkout library only contains a single entrypoint (`@spartacus/checkout`), which contains all checkout flows. On the other hand, in Spartacus 5.0, the library has been further decoupled into different business logic features, with the intention of having a smaller bundle size by splitting it into three different entrypoints instead (`@spartacus/checkout/base/root`, `@spartacus/checkout/b2b/root`, or `@spartacus/checkout/scheduled-replenishment/root`).
+
+If your application is already configured to load from `@spartacus/checkout/root`, you would need to update the reference that imports from it by choosing from either of the three different entrypoints as mentioned earlier.
```

Comment:
Not really. B2B and Repl both require Base. 
Moreover, Repl requires B2B.

Reply:
gotcha, I forgot 😓